applications/StructuralMechanicsApplication/custom_utilities/shell_cross_section.cpp

```diff
@@ -15,1258 +15,1473 @@
 #define M_PI 3.1415926535897932384626433832795
 #endif // M_PI
 
-

```

Comment:
what are the changes with respect to the previous version? this is also used in other elements, so are u sure your changes do not affect them?